spec/a11y/application/devise_spec.rb

```diff
@@ -24,37 +23,25 @@
     context '/mon_compte/informations' do
       before { visit '/mon_compte/informations' }
 
-      it do
-        is_expected.to be_accessible
-        is_expected.to have_skiplinks_ids
-      end
+      it { is_expected.to be_accessible }
     end
 
     context '/mon_compte/mot_de_passe' do
       before { visit '/mon_compte/mot_de_passe' }
 
-      it do
-        is_expected.to be_accessible
-        is_expected.to have_skiplinks_ids
-      end
+      it { is_expected.to be_accessible }
     end
 
     context '/mon_compte/antenne' do
       before { visit '/mon_compte/antenne' }
 
-      it do
-        is_expected.to be_accessible
-        is_expected.to have_skiplinks_ids
-      end
+      it { is_expected.to be_accessible }
     end
 
     context '/mon_compte/referents/:expert_id/domaines' do
       before { visit "/mon_compte/referents/#{expert.id}/domaines" }
 
-      it do
-        is_expected.to be_accessible
-        is_expected.to have_skiplinks_ids
```

Comment:
on peut plus le tester ?

Reply:
si le test est encore là 
`it { is_expected.to be_accessible }`

Reply:
`have_skiplinks_ids` est inclus dans `be_accessible` qui a été modifié et mis à jour avec une gem maintenue. C'est dans spec/support/axe/matchers/be_accessible.rb

Reply:
haan ok, ça y est, j'ai compris

Reply:
J'ai repris le nom `be_accessible` mais c'etait le matcher de l'ancienne gem qui regardait l'accessibilité mais qui n'est plus maintenue